### PR TITLE
[INJI-249]fix:Back button from the ellipsis page not working on the mobile

### PR DESCRIPTION
**Pull request by srikanth716**
https://github.com/tw-mosip/inji/assets/97477121/c4500c34-9495-4c0e-8e02-322d96d3bd6e

### CODE REVIEW

File: components/KebabPopUp.tsx [hunk 1]
```diff
@@ -21,9 +21,10 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
         type={props.iconType}
         color={Theme.Colors.GrayIcon}
       />
-      <BottomSheet
+      <Overlay
```
Comment (vharsh) [inline components/KebabPopUp.tsx:24]: You seem to have changed the UX here^, please keep screenshots handy and paste it in PR descriptions & Jira ticket.

^ I'm taking a guess here by referring to https://reactnativeelements.com/docs/2.3.2/bottomsheet

**Review by vharsh — COMMENTED** [review]
Please see if there's a change in UX design or if this code change will make a change in current UX.

**Comment by srikanth716** [comment]
> Please see if there's a change in UX design or if this code change will make a change in current UX.

there are no changes
,
Since the bottom sheet doesn't support closing the popup I have changed it to overlay as per the requirement

File: components/KebabPopUp.tsx (hunk 1 above)
Comment (srikanth716) [reply components/KebabPopUp.tsx:24]: I will attach the screen record for in PR descriptions & Jira ticket.

**Review by vharsh — APPROVED** [review]
lgtm, please attach a screenshot

**Review by tilak-puli — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]
Looks good to me.
If any bug is being reported for some UI effect, we can look for BackHandler for BottomSheet.
https://github.com/gorhom/react-native-bottom-sheet/issues/556

For now, we can close this with Overlay

**Pull request merged**